### DISCUSSION TITLE
[Spring] Blocking I/O 구간을 Reactor로 처리하기

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#389] [1. 개발 관련 글 모음]
Blocking I/O가 발생하는 구간을 Reactor를 사용해 처리하는 패턴에 대해 정리해봤습니다.
[블로그](https://blog.naver.com/ds4ouj/223392993576)

**Comment by 02ggang9** [comment]
글 잘 봤습니다~~!! 감사합니다
